src/spikeinterface/widgets/potential_merges.py

```diff
@@ -0,0 +1,244 @@
+            self.axes_amplitudes = [ax_amplitudes_ts, ax_amplitudes_hist]
+            row_offset += ncols + 1
+            self.axes_cc = []
+            for i in range(0, n_units):
+                for j in range(0, n_units):
+                    self.axes_cc.append(
+                        self.figure.add_subplot(
+                            self.gs[
+                                row_offset + (unit_len_in_gs) * i : row_offset + (unit_len_in_gs) * (i + 1),
+                                j * unit_len_in_gs : (j + 1) * unit_len_in_gs,
+                            ]
+                        )
+                    )
+            self.axes_cc = np.array(self.axes_cc).reshape((n_units, n_units))
+        self.previous_num_merges = len(merge_units)
+
+    def _update_plot(self, change=None):
+
+        merge_units = self.unit_selector.value
+        sorting_analyzer = self.data_plot["sorting_analyzer"]
+        channel_locations = sorting_analyzer.get_channel_locations()
+        unit_ids = sorting_analyzer.unit_ids
+
+        # unroll the merges
+        unit_ids_str = [str(u) for u in unit_ids]
+        plot_unit_ids = []
+        for m in merge_units.split("-"):
+            plot_unit_ids.append(unit_ids[unit_ids_str.index(m)])
+        self._update_gs(plot_unit_ids)
+
+        backend_kwargs_mpl = default_backend_kwargs["matplotlib"].copy()
+        backend_kwargs_mpl.pop("axes")
+        backend_kwargs_mpl.pop("ax")
+
+        amplitude_data_plot = self.w_amplitudes.data_plot.copy()
+        amplitude_data_plot["unit_ids"] = plot_unit_ids
+        self.w_amplitudes.plot_matplotlib(amplitude_data_plot, ax=None, axes=self.axes_amplitudes, **backend_kwargs_mpl)
+
+        unit_template_data_plot = self.w_templates.data_plot.copy()
+        unit_template_data_plot["unit_ids"] = plot_unit_ids
+        unit_template_data_plot["same_axis"] = True
+        unit_template_data_plot["set_title"] = False
+        unit_template_data_plot["scale"] = self.scaler.value
+        unit_template_data_plot["widen_narrow_scale"] = self.widen_narrow.value
+        # update templates and shading
+        templates_ext = sorting_analyzer.get_extension("templates")
+        unit_template_data_plot["templates"] = templates_ext.get_templates(unit_ids=plot_unit_ids, operator="average")
+        unit_template_data_plot["templates_shading"] = self.w_templates._get_template_shadings(
+            plot_unit_ids, self.w_templates.data_plot["templates_percentile_shading"]
+        )
+        self.w_templates.plot_matplotlib(unit_template_data_plot, ax=self.ax_templates, axes=None, **backend_kwargs_mpl)
+        self.ax_templates.axis("off")
+        self.w_templates._plot_probe(self.ax_probe, channel_locations, plot_unit_ids)
+        crosscorrelograms_data_plot = self.w_crosscorrelograms.data_plot.copy()
+        crosscorrelograms_data_plot["unit_ids"] = plot_unit_ids
```

Comment:
There is a bug here. It should be more

`index = np.flatnonzero(np.isin(self.unique_merge_units, plot_unit_ids))
        data = crosscorrelograms_data_plot["correlograms"]
        data = data[index][:, index]
        crosscorrelograms_data_plot["correlograms"] = data`

Otherwise, the CC plots are never updated

Reply:
Makes sense! Can you push the fix?

Reply:
Maybe call `data` -> `updated_correlograms`

Reply:
Note sure I can push in your branch, can I?